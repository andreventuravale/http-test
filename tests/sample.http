@@host=https://jsonplaceholder.typicode.com
@@dash={{$guid}}

# @name sample1
# @ignoreHeaders .*
# @expect status 200
# @expect statusText OK
GET {{host}}/todos/1?_={{dash}}
content-type: application/json

###

# @name sample2
# @ignoreHeaders .*
# @expect status 200
# @expect statusText OK
GET {{host}}/todos/2?_={{dash}}
content-type: application/json
