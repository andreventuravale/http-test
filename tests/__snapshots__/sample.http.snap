// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/home/avale/node/jest-dot-http-files/tests/sample.http GET https://jsonplaceholder.typicode.com/todos/1?_=2d29d515-3a78-4a0f-afaf-7db768444441 1`] = `
{
  "request": {
    "body": "undefined",
    "headers": [
      [
        "content-type",
        "application/json",
      ],
    ],
    "meta": {
      "expect": [
        [
          "status",
          "200",
        ],
        [
          "statusText",
          "OK",
        ],
      ],
      "ignoreHeaders": {
        "global": false,
        "value": ".*",
      },
      "name": {
        "global": false,
        "value": "sample1",
      },
    },
    "method": "GET",
    "url": "https://jsonplaceholder.typicode.com/todos/1?_=2d29d515-3a78-4a0f-afaf-7db768444441",
    "variables": {
      "dash": {
        "global": true,
        "value": "2d29d515-3a78-4a0f-afaf-7db768444441",
      },
      "host": {
        "global": true,
        "value": "https://jsonplaceholder.typicode.com",
      },
    },
  },
  "response": {
    "body": {
      "completed": false,
      "id": 1,
      "title": "delectus aut autem",
      "userId": 1,
    },
    "headers": [
      [
        "access-control-allow-credentials",
        Anything,
      ],
      [
        "alt-svc",
        Anything,
      ],
      [
        "cache-control",
        Anything,
      ],
      [
        "cf-cache-status",
        Anything,
      ],
      [
        "cf-ray",
        Anything,
      ],
      [
        "connection",
        Anything,
      ],
      [
        "content-encoding",
        Anything,
      ],
      [
        "content-type",
        Anything,
      ],
      [
        "date",
        Anything,
      ],
      [
        "etag",
        Anything,
      ],
      [
        "expires",
        Anything,
      ],
      [
        "nel",
        Anything,
      ],
      [
        "pragma",
        Anything,
      ],
      [
        "report-to",
        Anything,
      ],
      [
        "reporting-endpoints",
        Anything,
      ],
      [
        "server",
        Anything,
      ],
      [
        "transfer-encoding",
        Anything,
      ],
      [
        "vary",
        Anything,
      ],
      [
        "via",
        Anything,
      ],
      [
        "x-content-type-options",
        Anything,
      ],
      [
        "x-powered-by",
        Anything,
      ],
      [
        "x-ratelimit-limit",
        Anything,
      ],
      [
        "x-ratelimit-remaining",
        Anything,
      ],
      [
        "x-ratelimit-reset",
        Anything,
      ],
    ],
    "status": 200,
    "statusText": "OK",
  },
}
`;

exports[`/home/avale/node/jest-dot-http-files/tests/sample.http GET https://jsonplaceholder.typicode.com/todos/2?_=2d29d515-3a78-4a0f-afaf-7db768444441 1`] = `
{
  "request": {
    "body": "undefined",
    "headers": [
      [
        "content-type",
        "application/json",
      ],
    ],
    "meta": {
      "expect": [
        [
          "status",
          "200",
        ],
        [
          "statusText",
          "OK",
        ],
      ],
      "ignoreHeaders": {
        "global": false,
        "value": ".*",
      },
      "name": {
        "global": false,
        "value": "sample2",
      },
    },
    "method": "GET",
    "url": "https://jsonplaceholder.typicode.com/todos/2?_=a7392bf0-74e4-4aa8-8db7-57a6251e8d58",
  },
  "response": {
    "body": {
      "completed": false,
      "id": 2,
      "title": "quis ut nam facilis et officia qui",
      "userId": 1,
    },
    "headers": [
      [
        "access-control-allow-credentials",
        Anything,
      ],
      [
        "alt-svc",
        Anything,
      ],
      [
        "cache-control",
        Anything,
      ],
      [
        "cf-cache-status",
        Anything,
      ],
      [
        "cf-ray",
        Anything,
      ],
      [
        "connection",
        Anything,
      ],
      [
        "content-encoding",
        Anything,
      ],
      [
        "content-type",
        Anything,
      ],
      [
        "date",
        Anything,
      ],
      [
        "etag",
        Anything,
      ],
      [
        "expires",
        Anything,
      ],
      [
        "nel",
        Anything,
      ],
      [
        "pragma",
        Anything,
      ],
      [
        "report-to",
        Anything,
      ],
      [
        "reporting-endpoints",
        Anything,
      ],
      [
        "server",
        Anything,
      ],
      [
        "transfer-encoding",
        Anything,
      ],
      [
        "vary",
        Anything,
      ],
      [
        "via",
        Anything,
      ],
      [
        "x-content-type-options",
        Anything,
      ],
      [
        "x-powered-by",
        Anything,
      ],
      [
        "x-ratelimit-limit",
        Anything,
      ],
      [
        "x-ratelimit-remaining",
        Anything,
      ],
      [
        "x-ratelimit-reset",
        Anything,
      ],
    ],
    "status": 200,
    "statusText": "OK",
  },
}
`;
